packages/kit/test/apps/options/test/test.js

```diff
@@ -7,9 +7,11 @@ test.describe.configure({ mode: 'parallel' });
 
 test.describe('base path', () => {
 	test('serves a useful 404 when visiting unprefixed path', async ({ request }) => {
-		const response = await request.get('/');
+		const response = await request.get('/slash/', { headers: { Accept: 'text/html' } });
```

Comment:
Why did this need to be changed? Is there anything different about how `/` works in this scenario now?

Reply:
ah, okay — seems it redirects to `base` automatically in the `/` case

Reply:
The dev server will automatically redirect you to the base path rather than giving the error message

Reply:
did you want to remove the `Accept` header here?

Reply:
did you want to remove the `Accept` header here?
```suggestion
		const response = await request.get('/slash/');
```

Reply:
That would change the response, so no. Though I guess we could test both HTML and plain responses